mbf_abstract_nav/src/abstract_controller_execution.cpp

```diff
@@ -254,18 +254,24 @@ bool AbstractControllerExecution::reachedGoalCheck()
 
 bool AbstractControllerExecution::cancel()
 {
-  // request the controller to cancel; it returns false if cancel is not implemented or rejected by the plugin
-  if (!controller_->cancel())
+  // Request the controller to cancel; returns true if he takes care of stopping, returning CANCELED when done,
```

Comment:
controller seem to be very male in this pr :grin:

Reply:
mmm....  I think I refer always as "him"...  :thinking: 
I'll start using it